config/src/config/state_sync_config.rs

```diff
@@ -435,7 +435,7 @@ impl Default for AptosDataClientConfig {
             data_multi_fetch_config: AptosDataMultiFetchConfig::default(),
             ignore_low_score_peers: true,
             latency_filtering_config: AptosLatencyFilteringConfig::default(),
-            latency_monitor_loop_interval_ms: 100,
+            latency_monitor_loop_interval_ms: 10,
```

Comment:
What is this change? Is this required in this PR?

Reply:
This is actually meant to improve the accuracy of state sync metric collection. Not relevant to this particular PR. So, removing it.